### ISSUE TITLE
AttributeError: 'Client' object has no attribute 'jobs'

### ISSUE THREAD

**Issue by vmorozov** [GoogleCloudPlatform/gcloud-python#1287]
I try to get of list of finished bigquery jobs assuming like https://cloud.google.com/bigquery/docs/reference/v2/jobs/list . Is it  correct method?

```python
from gcloud import bigquery
client = bigquery.Client(project='tdi-genome-1')
jobs = client.jobs()
AttributeError: 'Client' object has no attribute 'jobs'
```

Version info:

```python
print(client._connection_class.USER_AGENT)
gcloud-python/0.8.0
```

**Comment by tseaver** [comment]
PR #1260 will add the `client.list_jobs` method when it is merged.

**Comment by tseaver** [comment]
Oops, sorry.  #1260 is the issue for adding the method;  #1279 is the PR which will close that issue.

**Comment by tseaver** [comment]
Closed via #1279.
